engine/sound/src/sound_null.cpp

```diff
@@ -25,6 +25,11 @@ namespace dmSound
     {
         char* m_Buffer;
         uint32_t m_BufferSize;
+
+        // make sure it's the same size for both 32/64 bit. Makes it easier for tests
+        #if !(defined(_WIN64) || defined(__x86_64__) || defined(__ppc64__))
+        uint64_t _pad;
+        #endif
```

Comment:
The issue was only on 32bit system, and I was using 64 bit, so I didn't see it first.
(The previous issue was on 64 bit system too, so I got a bit sidetracked)

Also, the output from the dmLogXxx functions seem to arrive before the printf's, so it wasn't obvious what the problem was. Another sidetrack.

Reply:
hmmm... It seems asserting based of sizeof(SoundData) was more a can of worms than anything else :-/